package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "strip-ansi",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
Please don't bump the package version. The maintainers will do that when they release a new version.